tests/Feature/Notifications/NotificationTableNamingTest.php

```
@@ -0,0 +1,68 @@
+        public function via( object $notifiable ): array
+        {
+            return ['database'];
+        }
+
+        public function toArray( object $notifiable ): array
+        {
+            return ['message' => 'hello'];
+        }
+    } );
+
+    expect( Notification::count() )->toBe( 1 )
```

PHPStan (2.2.7)

[error] Call to an undefined static method ArtisanPackUI\CMSFramework\Modules\Notifications\Models\Notification::count().


(staticMethod.notFound)